src/matrixHelpers/FormioScript/scriptProd.js

```diff
@@ -0,0 +1,44 @@
+import { createScripts } from ".";
+
+const version = window.formioQldCdnVersion || "v1/v1.x.x-latest";
```

Comment:
could this be in its own file loaded into all scripts so its not replaced in 4 places on update.